Conceptual_Design.md

EWinnie opened the pull request.

EWinnie committed:
Create Conceptual_Design.md

EWinnie committed:
Update Conceptual_Design.md

Just putting something in here. Format and add to it later

KamdenE committed:
Create Conceptual Design.md

Created Folder for organizing.

KamdenE committed:
Update Conceptual Design.md to original md.

KamdenE committed:
Delete Conceptual_Design.md

KamdenE committed:
Updated Conceptual Design.md to a better format

EWinnie committed:
Update Conceptual Design.md

A whole bunch of hooplah

crgraves8 committed:
Update Conceptual Design.md

Added "Hard wired Communication" section. I still need to format it correctly.

crgraves8 committed:
Update Conceptual Design.md

Capitalization error

EWinnie committed:
Update Conceptual Design.md

Adding things from the google doc as well as statement of contribution

KamdenE committed:
Updated Budget in Conceptual Design.md

KamdenE committed:
Updated Budget in Conceptual Design.md

Changed Microcontroller specification.

KamdenE committed:
Updated Budget in Conceptual Design.md

Fixed the pricing.

KamdenE committed:
Added Gantt chart to Conceptual Design.md

EWinnie committed:
Update Conceptual Design.md

drive train

CooperM42 committed:
Update Conceptual Design.md

Add my current statement of contribution.

dtran143 commented:
Good draft for the conceptual design. Some things to note are spacing of certain beginning sentences. It is missing a hardware block diagram.

Jdcaten42 commented:
I like how the conceptual design is coming along. 
A few things to note:
-there is a random tab placed in the introduction.
-You probably should mention Circe soft in the introduction rather than the formulation of the problem.
the comparison and analysis of potential solutions was done on a very low level. -you don't need to explain every choice and option, just give a brief explanation why. 
-Your goal is to make a robot that can operate in contested environments, which means a hostile adversary may intend to blow it up. The robot should be somewhat disposable, and the way you structure your budget should show this. Organize the budget in two categories, repeating costs (cost to make the device) and onetime costs (testing and software).
-You need a negative in the ethical considerations. ex: may make it easier for institutions to rationalize the use of robots to kill.
-for the list of potential solutions, you have a high-end processor that has clear visual and audio capabilities over the specified range of the device and it comes for free (the human operator).
-You are dealing with a very short range, might it be better to use the local Cartesian plane coordinate system or does your customer intend to increase this range potentially?
You are missing a hardware block diagram, but given how extensive your comparison of different solutions is, your diagram is going to look like: design a battery- specifics on how to do that. Design an autonomous navigation system to execute a series of commands, a navigation system using gps, and a navigation system by user. - and specifics on how to do that.
if that is not what the hardware block diagram is going to be, then you probably need to reiterate your problem and solution.

krizithy commented:
Lots of good information here. The organization flows and reads well. Missing block diagram(s) for certain section(s). Keep it up

93marisol commented:
Solid, I think the formatting needs less bulletin point especially on sections that are just one bulletin point. I also noticed that you are still needing hardware block diagram and operational flow chart.

ambacon43 commented:
The specifications and constraints are incredibly well laid out, nice work. My only suggestion there would be to take another look at the "Additional Design Constraints" section, because most of those sound more like specs to me. The comparative analysis section is also awesome, super well organized & easy to follow! I love how you broke up the solutions for the three major challenges of this project; it looks like a lot of thought and research went into that. I wonder if in the chosen solution section, you guys may want to leave a little wiggle room in case you change your mind on which solutions you use? But if you are pretty confident in your plan, it may not matter. 

Other suggestions: 1) don't use your names in the references- I don't know if that was just a temporary thing, but it looks like you're referencing yourselves. 2) Hardware Block Diagram & Operational Flow Chart. 3) I like the high level overview, but you may need to designate specific tasks to each team member even though it will be collaborative. Overall, great job, it's looking really good so far!

crgraves8 committed:
Update Conceptual Design.md

Just cleaned up a bit. Took out names from the citations, deleted the indentations where needed to make the overall look more aesthetic, and other random small changes.

KamdenE committed:
Added Flow Chart and Block Diagram

KamdenE committed:
Updated Diagrams in Conceptual Design.md

gwmarcrum committed:
Update Conceptual Design.md

Added SoC and References

EWinnie committed:
Update Conceptual Design.md

Nothing major

macky-mack commented:
For a first draft of the conceptual is a great start. People are able to read it in depth and understand the overall design work on paper before seeing the diagram. I would say the formatting is cohesive with ever section. I have nothing major to say, it is a great start on the work.

dtran143 reviewed (approved):
At the time of reading, I had some issues with the Gantt chart loading but I don't know if that's on my end. However, excellent conceptual design, there's nothing I can really give critique on. Maybe some more bullet points below bullet points expanding on what was stated but that would be it.

m7md023x commented:
I have nothing to add, but it's a good idea to see the 6 questions at the end of the conceptual design PowerPoint and compare it to your design to improve it.

jmdavis43 reviewed (approved):
This conceptual design looks great and is very informative. My only advice is maybe when a bulleted sentence is two lines long indent both lines to show that it is one sentence.

Jdcaten42 commented:
This is ready to be moved on. I will say that the document is long, but if you believe it is needed, I see no reason to make changes.

Henry100202 reviewed (approved):
Looks great to me, nothing to add that hasn't been said.

MaddoxCagle commented:
Looks good, is 290W a lot of power or am I just thinking it is lol?

CooperM42 committed:
Update Conceptual Design.md

Edited Drivetrain/Motor and the corresponding reference.

KamdenE committed:
Updated Gantt chart links in Conceptual Design.md

KamdenE committed:
Updated Statement of Contribution in Conceptual Design.md

macky-mack reviewed (approved):
This design is really detailed and can see that you all really listened to what the customer was asking for and expanded more on it to make it better. Everything looks great I have no complaints about it. Good job guys!

jmdavis43 reviewed (approved):
What is an ad hoc RF workaround? And is the section right above Design requirements and constraints not supposed to be in that section? For the robot chassis and cable laying device is it necessary for them to be in the high-level solution if they are just there to say this is up to the mechanical engineering team? If you do not see a need in fixing any of these just ignore because overall this is a solid conceptual design.

Hammy2017 reviewed (approved):
I believe you guys have mostly killed it with this one, but there is one small thing. That being that there is no explicitly labeled "Atomic Subsystem Specification" section but your higher solution section kind of fills that role

dtran143 commented:
I believe your conceptual design is very complete in all aspects. Some optional things that can be added are the format in which the real time data is transmitted like that protocol and what receiving format. Things like what the user can do for certain error codes could be another aspect to talk about.

BretBall commented:
Y'all didn't request my review. Bums.

Introduction
- good

Fully Formulating the Problem
- is the use of "should" ok here? probably.
- 10 Hz data transmission speed? Most ethernet is 10 or 100 Mbps.
- missing shall word(s) in additional design constraints. maybe

Comparitive Analysis of Solutions
- kobalt 80v might be a bangin battery for this. Rips on a weedeater
- good 

High-Level Solution Overview
- by using the power distribution module to distribute different amounts of power. Paragraph needs rework.
- DC transformer?
- All of the motor portion should be rewritten since cooper did it.

Diagram and flowchart
- hard
- flowchart does seem a little off but I'm a tweaker.

EPSC
- hard

Long Title
- hard

Constraints
- hard

Resources
- ghant chart hard
- that seems super cheap
- Cooper is proficient in c++?

Statement of Contribution
- contributions?
- use periods or commas not both.

Summary: This report is tight. Everything I said can probably be ignored like the last time. Got a little hard to read at the end due to the tussin.

tylerbare0 commented:
**PROS:**
Introduction
- Quick and to the point, me like

Budget
- Looks great as always :)

Diagrams:
- The diagrams are sufficient for conceptual design in my opinion. Good job Kamden (*leans over and kisses you on the forehead*)

**CONS:**
Fully Formulating the Problem
- Maybe be more specific when stating the limitations of RF communications and how CIRCE addresses these issues?

Comparative Analysis of Solutions
- Power Supply Analysis: I would list different subsections under the different types of batteries that are listed and state the pros and cons. Have y'all looked into Lithium Polymer (LiPo) batteries (which I guess would be a subset of Lithium Ion batteries)? They are extremely lightweight.

Ethical, Professional, and Standards Considerations
- Y'all listed several great ethical considerations but maybe provide more details on how CIRCE complies with relevant standards and ethics, like possibly expanding the broader implications for civilian applications section?

tylerbare0 commented:
Also y'all did not request me to review :(

ellerenfroe commented:
Great job! I like how the potential solutions are split up and formatted with different solutions for each section. The only thing I noticed was that the atomic subsystem section needs to be formulated in shall statements. Other than that, it looks good!

Henry100202 commented:
I like how you left no room for confusion in the Formulating the Problem section with the "The customer, DEVCOM, has specified" statements. However, the high-level solutions overview might need to be rewritten. I just feel like it needs more detail for a "detailed solution". Otherwise it's good. Also, I was not requested to review, sadge :(

93marisol commented:
Formulating the Problem and Comparative Analysis are complete sections, good job. My question would be, you have a BMS and it's monitoring battery health while on the field, but your main objective is getting the robot from point A to point B... does battery health have any influence on proceedure at all while deployed?

Jdcaten42 commented:
It looks good, I will say that you need to show the autonomous behavior in the flow chart but this is ready to move on.

ambacon43 commented:
Great job. The reasoning behind your selections from the potential solutions is explained thoroughly. My one suggestion is standardizing the formatting a bit. For example, in the high level solution, the power system section has spaces between the bullet points and some words emboldened, while the other sections don't. Not a huge deal of course, but tweaking little things like that will give your report a cleaner, more finished look.

MaddoxCagle reviewed (commented):
"CIRCE consists of three primary components: CirceBot, the robotic platform responsible for physically deploying the cable; CirceSoft, the path-planning software that directs CirceBot’s movements; and a hardline communication interface, which enables telemetry transmission and control without relying on RF signals." This sentence feels like a run on, not to be the grammar police, and I may be wrong... I think everything flows well, and I might add color to the hardware and flow chart, but I think everything else has been said.

krizithy reviewed (approved):
Looks great

micahr234 commented on Conceptual Design/Conceptual Design.md:
```diff
@@ -0,0 +1,308 @@
+
+
+## Design Requirements and Constraints
+
+### Communication Specifications
+- CirceBot shall be designed to receive and execute commands in the format supplied in the CirceSoft2CirceBot.proto spec from CirceSoft and transmit telemetry data in the CirceBot2CirceSoft.proto spec to CirceSoft.
+- CirceBot shall accept planned paths in the specified format and follow those paths to its objective.
+- **Customer Requirement**: The customer, DEVCOM, has specified that CirceBot shall use their provided software (CIRCESOFT) and format for data telemetry processing and communication.
+
+### Cable Deployment Standards
+- CirceBot shall dispense cable according to installation guidelines, including proper curve radiuses, strain relief, and tension tolerances.
+- **Relevant Standards:**
+  - **ANSI/TIA-568.2-D**: Minimum bend radius should be four times the cable's outer diameter.
+  - **ANSI/TIA-568.2-D**: Proper strain relief is required to prevent mechanical stress.
+  - **ANSI/TIA-568.2-D**: Maximum pulling tension should not exceed 25 pounds (110 Newtons).
+
+### Power Source Requirements
+- It shall be equipped with an independent power source capable of at least 20 minutes of operation.
+- The battery should maximize power while minimizing weight.
+- Example: Talentcell 12V 24Ah rechargeable deep-cycle lithium battery.
+
+### Telemetry and Navigation
+- CirceBot shall transmit real-time data including:
+  - Current position
+  - Current velocity
+  - Meters of cable left
+  - Heading
+  - Battery life percentage
+  - Error codes (if any)
+- Minimum data transmission speed: **10 Hz**
+- CirceBot shall receive Next Position waypoints and navigate accordingly.
+
+### Additional Design Constraints
+- CirceBot shall carry up to **100 yards (10 lbs) of Ethernet cable**.
+- Quick and tool-free replacement of Ethernet cable reels (within 2 minutes).
+- Shall operate both **with and without GPS assistance**.
+- Shall use minor obstacle avoidance mechanisms.
+- **Relevant Safety Standard:** ANSI/RIA R15.08 (Obstacle Detection and Avoidance for AMRs).
+
+---
```
I keep reading the same thing multiple time. Please clean this up. Also clearly delineate which requirements are specification from your customer and which are constraints from other sources.

micahr234 commented on Conceptual Design/Conceptual Design.md:
```diff
@@ -0,0 +1,308 @@
+
+### Nickel-Metal Hydride (NiMH) Battery:
+NiMH batteries offer a middle ground between lithium-ion and lead-acid batteries in terms of cost and performance. They are lighter than lead-acid batteries and provide a decent amount of power, but their energy density is lower than lithium-ion batteries. While NiMH could work for CIRCE's needs, it would not provide the same level of runtime or efficiency as lithium-ion batteries, especially whe ...
+
```
How are you going to couple to the ethernet spool while it is rotating? Did you evaluate options for this?

Why have the spool on the robot?

What will be the brains of the robot? Compare different platforms... which do you think is the best solution?

Do other people already make these robots? What can you learn from them? https://www.youtube.com/watch?v=2yponSiLaBQ

Would a live feed video be of value?

KamdenE replied:
Micah,

We are evaluating options for the spool with the ME team and we do believe a system with the rotating Ethernet port will work. We will need to pick that up in the proposal here shortly.

We still need the spool on the robot if it is to place the cable on open ground and be able to "staple" it if it needs to make a turn. While the idea of having the spool be on the ground like the video showed work well, I do not believe it would satisfy what DEVCOM is preferring the cable act like as the robot would only be able to go straight or have to pull the cable through larger staples and drag it. 

While we have the Arduino 2560 as our current option, Dr. Van Neste brought up that we look into a Teensy as they respond much faster than arduinos and are priced the same.

I do not think currently that a live video be of value if the robot is to control itself and we need to conserve power. I have seen some videos with cars using LIDAR and being able to see that on a screen so maybe I can look into that for this robot. 

Thank you for the advice and we have lots of things to now look into.

micahr234 replied:
Teensy does not look like it can talk ethernet. Can you compare CPU options?

micahr234 replied:
Also I know that you do not need live video but think about how useful it might be. For example if the bot gets stuck maybe a human can get it unstuck by controlling it over the live feed. I will leave this decision entirely up to you and will not dock you in any way for not doing a live feed. I just wanted to express my opinion.

EWinnie committed:
Update Conceptual Design.md

Fixed the repetitive stuff because I think that got overlooked when we reformatted the document at the beginning. Also, I addressed the spooling comment, and the "do other people already make something" which I stuck in resources

EWinnie committed:
Update Conceptual Design.md

Finishing up revisions

micahr234 commented on Conceptual Design/Conceptual Design.md:
```diff
@@ -0,0 +1,291 @@
+
+### Communication Specifications
+- **Customer Requirement**: The customer, DEVCOM, has specified that CirceBot shall use their provided software (CIRCESOFT) and format for data telemetry processing and communication. This software is either provided by DEVCOM or the computer science team.
+- Live video feed will not be provided. At this stage in development the robot is to be mainly autonomous with little user interaction; therefore, the relaying of it's exact position shall be proficient. This could potentially be added on later, but the main concern would be power consumption. 
```
proficient????

micahr234 commented on Conceptual Design/Conceptual Design.md:
```diff
@@ -0,0 +1,291 @@
+The ethernet cable from the spool will connect to the robot via a swivel connection point to allow for the spool to rotate without compromising the integrity of the cable from twists or bends. If the cable is twisted too many times it could degrade the signals that are being sent and received. Furthermore, the swivel adapter that will most likely be used will alos provide a length of cable along  ...
+
+# Chosen Solution
+After careful evaluation, LiDAR was selected for the obstacle avoidance system due to its superior accuracy, long-range detection, and adaptability to varying environmental conditions. Its ability to provide reliable 360-degree mapping makes it the best fit for CIRCE’s complex navigation needs. Similarly, the ECEF coordinate system was chosen for its global consistency, accuracy, and suitability  ...
```
I think you are making the wrong decision with lidar (depth camera would be better) but this choice is entirely up to you.

micahr234 commented on Conceptual Design/Conceptual Design.md:
```diff
@@ -0,0 +1,291 @@
+# High-Level Solution Overview
+Given the complexity of this project and the need for strong teamwork, a divide-and-conquer approach is most effective. The robot is structured into distinct subsystems, allowing the team to analyze how each component integrates into the whole. While some subsystems are more simple, others demand more intricate solutions. Each subsystem is assigned to a team member with either an interest or expe ...
+## **Power System**
+- Runs on a **lithium-ion battery** (Exact Model TBD). This battery will power the entire robot, which includes the tracks and motors, the microcontroller, all sensors required, operating systems and the cable laying device. 
```
You need to be more specific. How much energy should the battery pack store?

micahr234 commented on Conceptual Design/Conceptual Design.md:
```diff
@@ -0,0 +1,291 @@
+Given the complexity of this project and the need for strong teamwork, a divide-and-conquer approach is most effective. The robot is structured into distinct subsystems, allowing the team to analyze how each component integrates into the whole. While some subsystems are more simple, others demand more intricate solutions. Each subsystem is assigned to a team member with either an interest or expe ...
+## **Power System**
+- Runs on a **lithium-ion battery** (Exact Model TBD). This battery will power the entire robot, which includes the tracks and motors, the microcontroller, all sensors required, operating systems and the cable laying device. 
+- Requires **step-up/step-down transformers** for different power needs. To accomplish running everything off of one battery it is safe to assume that step-up/step-down transformers will be required. Not every single atomic subsystem will require the same power as the next; therefore, multiple transformers will be needed. This could be solved with a PDM (power distribution module). This will allo ...
```
You need to give specific voltages. Do not worry about being wrong. You can change things latter on. But you should have some idea of the voltages and currents that the power supply needs to deliver.

micahr234 commented on Conceptual Design/Conceptual Design.md:
```diff
@@ -0,0 +1,291 @@
+- Runs on a **lithium-ion battery** (Exact Model TBD). This battery will power the entire robot, which includes the tracks and motors, the microcontroller, all sensors required, operating systems and the cable laying device. 
+- Requires **step-up/step-down transformers** for different power needs. To accomplish running everything off of one battery it is safe to assume that step-up/step-down transformers will be required. Not every single atomic subsystem will require the same power as the next; therefore, multiple transformers will be needed. This could be solved with a PDM (power distribution module). This will allo ...
+
+- **Battery Management System (BMS)** will monitor battery activity. This system should only be outputting power and receive no inputs unless being recharged. The battery will have a BMS (battery management system), where it will relay its status such as battery life (%), runtime, and other necessary information to ensure it’s health like temperature. 
```
Be more concrete and clear: BMS will transmit the state of charge, state of health, and battery temperature to the main controller.

Also will the system have a safety shutdown if the battery becomes too hot or excessive current is drawn?

micahr234 commented on Conceptual Design/Conceptual Design.md:
```diff
@@ -0,0 +1,291 @@
+| Unexpected | 50W |
+| **Total** | **290W** |
+
+- Based off the above numbers the battery should be **no less than 12V with a 9Ah rating**. 
```
I see you have addressed the battery size. Good. Maybe put this with the battery spec?????

EWinnie replied:
Please see line 29, 64, 122 and 139. I'm unsure of where else to put this without being overly redundant, which we got multiple comments on in peer review.

micahr234 commented on Conceptual Design/Conceptual Design.md:
```diff
@@ -0,0 +1,291 @@
+
+# High-Level Solution Overview
+Given the complexity of this project and the need for strong teamwork, a divide-and-conquer approach is most effective. The robot is structured into distinct subsystems, allowing the team to analyze how each component integrates into the whole. While some subsystems are more simple, others demand more intricate solutions. Each subsystem is assigned to a team member with either an interest or expe ...
+## **Power System**
```
For each subsystem formulate your requirements as shall statements.

micahr234 commented on Conceptual Design/Conceptual Design.md:
```diff
@@ -0,0 +1,291 @@
+- A computer will communicate to the ardruino through the ethernet cable using the port described above. This will allow for easier user interface and ease of transferring coordinates to the robot.
+
+## **Drive train/motors**
+- Two 12V 75W Brushless DC Motors each capable of running at a rated speed of 3000 rpm and with a rated torque of 0.24 N-m will be used to move the drivetrain of CirceBot[4]. This motor was chosen due to meeting the power consumption listed in the drivetrain portion of the Power System section above. This is the best estimate possible given the current understanding. 
```
Check with your mechanical counterparts to make sure they are planning to use these motors.

micahr234 commented on Conceptual Design/Conceptual Design.md:
```diff
@@ -0,0 +1,291 @@
+
+
+## **Cable Laying Device**
+- Again this will likely be left up to the Mechanical Engineering team, so that they can better develop the correct materials and hardware to ensure a durable and capable system.
```
You do not have to list the mechanical teams systems. I just need enough explanation to understand how your controls are going to operate.

micahr234 commented on Conceptual Design/Conceptual Design.md:
```diff
@@ -0,0 +1,291 @@
+
+## **Cable Laying Device**
+- Again this will likely be left up to the Mechanical Engineering team, so that they can better develop the correct materials and hardware to ensure a durable and capable system.
+
```
Where is the motor for you de-spooler?

micahr234 replied:
Where is the lidar camera system?

crgraves8 committed:
Update Conceptual Design.md

Changed the word proficient in that sentence, but I still don't know if I changed it to be what you are trying to say in that sentence @Evan

Added shall statements to most of the subsystem sections. (I did not touch the operating system/brain of robot section because I did not know the best way to formulate those into shall statements)(also did not touch drivetrain as this whole section will likely be changed whenever ME team sends us that kit they are getting)

Deleted the robot chassis section per Micah's request. 

Added a lidar camera/depth sensor subsystem. I left this blank for whoever knows what they are talking about. Confused on what to even put here because I feel like it will repeat what is already stated previously in the document but?

I put a note under cable laying device subsystem. I basically left this subsystem in just in case we decide we need a motor for the cable dispensing out of the box. 

I don't think that the division of labor should be changed, but I will let y'all look at it as well. Seems to be fine to list their strengths and then the subsystem they are in charge of based on that but idk.

EWinnie committed:
Update Conceptual Design.md

Fixed all power questions I believe and reformatted the ME collaboration part to make that flow and look better. 

I addressed the motor for the spooling.

I addressed the BMS.

Addressed specific voltage comment. Definitely wrong but we'll fix later.

KamdenE committed:
Updated CPU Choice and LiDAR Options in Conceptual Design.md

KamdenE committed:
Updated High Level Solution in Conceptual Design.md

gwmarcrum committed:
Update Conceptual Design.md

Added the information under the "Lidar Camera/Depth Sensor" section.

CooperM42 committed:
Update Conceptual Design.md

edited the drivetrain with ME info

micahr234 commented:
Sorry that I was not able to review on Wednesday. I got busy. If it impacts your grade I will make it right.

micahr234 commented on Conceptual Design/Conceptual Design.md:
```diff
@@ -0,0 +1,294 @@
+Given the complexity of this project and the need for strong teamwork, a **divide-and-conquer** approach shall be the most effective strategy. The robot shall be structured into distinct subsystems, allowing the team to analyze how each component integrates into the whole. While some subsystems shall be relatively simple, others shall require more intricate solutions. Each subsystem shall be assi ...
+## **Power System**
+- CirceBot shall run on a **lithium-ion battery** (Exact Model TBD, but an example will be a talentcell 12v 50Ah). This battery will power the entire robot, which includes the tracks and motors, the microcontroller, all sensors required, operating systems and the cable laying device. A lithium ion deep cycle battery
+- CirceBot shall require **step-up/step-down transformers** for different power needs. To accomplish running everything off of one battery it is safe to assume that step-up/step-down transformers will be required. Not every single atomic subsystem will require the same power as the next; therefore, multiple transformers will be needed. For example there will likely be 4 motors, one for each wheel ...
```
Transformers work with AC but you need DC.

micahr234 commented on Conceptual Design/Conceptual Design.md:
```diff
@@ -0,0 +1,294 @@
+### **Robot Chassis**
+  - All controls to motors, drivers, and wheels will be handled by the EE team. 
+## **Lidar Camera/Depth Sensor**
+-LiDAR and depth cameras can work together to enable autonomous navigation without relying on GPS by providing complementary spatial data. LiDAR creates detailed 3D point clouds that map the environment, helping with obstacle detection and localization, while depth cameras offer precise close-range object recognition and depth maps for finer detail. The integration of both sensors allows for effe ...
```
This does not read like a specification. Identify the shall statements that constrain the type of lidar which you will choose.

micahr234 commented on Conceptual Design/Conceptual Design.md:
```diff
@@ -0,0 +1,294 @@
+# Ethical, Professional, and Standards Considerations
+
+## **Broader Impacts**
+- **Minimized Physical Footprint**: Encourages trust in robotics for military and civilian applications.
```
"Minimized Physical Footprint" this make no sense in context.

micahr234 commented on Conceptual Design/Conceptual Design.md:
```diff
@@ -0,0 +1,294 @@
+
+The design of the CIRCE system is built on well-established standards and best practices that ensure safety, reliability, and environmental responsibility, all while meeting the tough challenges of contested operational areas. Our team reviewed military, industrial, and environmental guidelines to shape every design decision.
+
+Military standards such as MIL-STD-882 have set a high bar for reliability of system safety and risk management[2]. This standard has led us to include thorough testing, validation, and fail-safe mechanisms so that even under unpredictable battlefield conditions, the system performs reliably, a must when human lives are on the line.
```
give example of mechanisms you have implemented to meet MIL-STD-882

micahr234 commented on Conceptual Design/Conceptual Design.md:
```diff
@@ -0,0 +1,294 @@
+|                           | Unforeseen Expenses          | 1        | 210           | 157            |
+| **Subtotal**              |                            |          |               | 157            |
+| **Total Budget**          |                            |          |               | 1500           |
+- Personnel and Division of Labor
+  -   The following are the members of Team 1 and their skills. After comparing the skills available to the skills required for specific subsystems, the team members have been assigned to work on the subsystems they’re most suited for.
+    - Kamden: Proficient in AutoCAD and Revit. Also skilled in Fusion 360, soldering, troubleshooting, and programming. He will be in charge of Microcontroller Subsystem due to his prior knowledge and enthusiasm in that area.
+    - Wayne: Expertise in hardwiring and troubleshooting, with knowledge in power systems and programming. He will be in charge of Navigation due to his knowledge in programming.
+    - Evan: Experienced with batteries, wiring, organization, and customer interaction. He will be in charge of the power system due to prior knowledge of batteries.
+    - Cooper: Proficient in AutoCAD and C++, with basic knowledge of SmartPlant/Smart3D. He will be in charge of the drive train/motor, which will involve communication with the ME team.
+    - Connor: Skilled in hardwiring, design, and 3D modeling in Revit. He will be in charge of the hardwired communications.
+    - Both the Chassis subsystem and cable laying device will primarily be handled by the mechanical engineering team since that side pertains to materials more. It is intended for both teams to work closely together when needed. 
```
You need to assign each subsystem to a person.

KamdenE committed:
Updated Power System in Conceptual Design.md

Changed transformers to converters.

KamdenE committed:
Updated Lidar Section in Conceptual Design.md

KamdenE committed:
Updated Division of Labor in Conceptual Design.md

EWinnie committed:
Update Conceptual Design.md

Finished up review 3
Assigned atomic subsystems more clearly
Examples of standard on line 212
clarification and formatt of minor things.

KamdenE committed:
Updated Swivel Solution in Conceptual Design.md

KamdenE committed:
Update Power Systems in Conceptual Design.md

Second bullet will need some work still but updated the first one.

KamdenE committed:
Updated Resources in Conceptual Design.md
